### ISSUE TITLE
Tooltips look odd [NDS-19]

### ISSUE THREAD

**Issue by lucas-contreras** [wwnorton/design-system#138]
Hello, 

I updated to the latest Norton design system version (1.1.1),

"@wwnds/core": "1.1.1",
"@wwnds/react": "1.1.1",

After this update, I found the tooltips look a bit odd, the background is transparent and the content is over the arrow. I will attach an image.

![image](https://user-images.githubusercontent.com/6923007/116436144-7ded7300-a822-11eb-8041-e4db4533f77b.png)

[NDS-19]

[NDS-19]: https://wwnorton.atlassian.net/browse/NDS-19

**Comment by sh0ji** [comment]
Thanks for opening this, Lucas! I noticed that as well and from what I could tell, it was because of a build error of some kind. Does updating to [`v1.2`](https://github.com/wwnorton/design-system/releases/tag/v1.2.0) resolve it?

**Comment by lucas-contreras** [comment]
Hey Evan, sorry for the delay,

I updated to v1.2, the tooltips look good now, there's a minor thing that I just noticed, it seems the arrow is behind the tooltip box. It happens when the placement is right or left, for the top and bottom are displayed correctly. I attach an image below.

![image](https://user-images.githubusercontent.com/6923007/118289053-55fd4100-b4ab-11eb-9ef8-9f7b21d21e95.png)

Thanks!
